src/DockerDiskUsage.php

```diff
@@ -0,0 +1,542 @@
+<?php
+
+namespace BiffBangPow\DockerDiskUsage;
+
+use Colors\Color;
+use Docker\API\Model\ContainerInfo;
+use Docker\Docker;
+use Stringy\Stringy;
+
+class DockerDiskUsage
+{
+    const BYTES_TO_GIGABYTES = 1024 * 1024 * 1024;
+
+    const DETAILED_REPORT = true;
+    const SUMMARY_REPORT = false;
+
+    const FORMATTED_TEXT = true;
+    const PLAIN_TEXT = false;
+
+    const INCLUDE_MOUNTS = true;
+    const EXCLUDE_MOUNTS = false;
+
+    const NAME_WIDTH = 50;
+    const STATUS_WIDTH = 17;
+    const TABBED_WIDTH = 3;
+
+    /**
+     * @var Color
+     */
+    private $cliFormatter;
+
+    /**
+     * @var array
+     */
+    private $data = [];
+
+    /**
+     * @var Docker
+     */
+    private $docker;
+
+    /**
+     * @var array
+     */
+    private $errors = [];
+
+    /**
+     * @var array
+     */
+    private $images = [];
+
+    /**
+     * @var bool
+     */
+    private $includeMounts = self::EXCLUDE_MOUNTS;
+
+    /**
+     * @var bool
+     */
+    private $useTextFormatting = self::FORMATTED_TEXT;
+
+    /**
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->cliFormatter = new Color();
+        $this->docker = new Docker();
+    }
+
+    /**
+     * @param bool $levelOfDetail
+     * @return string
+     */
+    public function plainTextReport(bool $levelOfDetail = self::SUMMARY_REPORT): string
+    {
+        $this->getDockerData();
+        if ($levelOfDetail === self::DETAILED_REPORT) {
+            $output = $this->createDetailedOutput();
+        } else {
+            $output = $this->createSummaryOutput();
+        }
+        $output .= $this->appendErrors();
+        return $output;
+    }
+
+    /**
+     * @return string
+     */
+    public function helpInformation(): string
+    {
+        return '
```

Comment:
Can we use a HEREDOC? http://php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc